Rocket.Chat.ShareExtension/Base/SEXibInitializable.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  SEXibInitializable.swift
+//  Rocket.Chat.ShareExtension
+//
+//  Created by Matheus Cardoso on 3/9/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+protocol SEXibInitializable {
+    var contentView: UIView! { get set }
+    func initializeFromXib()
+}
+
+extension SEXibInitializable where Self: UIView {
+    func initializeFromXib() {
+        Bundle.main.loadNibNamed(String(describing: type(of: self)), owner: self, options: nil)
+
+        addSubview(contentView)
+        contentView.translatesAutoresizingMaskIntoConstraints = false
+
+        addConstraints(
+            NSLayoutConstraint.constraints(
```

Comment:
What do you think about using Layout Anchors instead of Visual Format Language? It avoids to use strings and makes the code more readable.